src/app/issuer/components/badgeclass-edit-form/badgeclass-edit-form.component.html

```diff
@@ -452,7 +452,6 @@ <h2 class="heading"><ng-container>{{ badgeCategory === 'competency' ? '4.' : '3.
 
 					<bg-formfield-markdown
 						[control]="badgeClassForm.rawControlMap.badge_criteria_text"
-						[newValue]="badgeClassForm.rawControlMap.badge_criteria_text.value"
```

Comment:
@jona159 One of the errors occurred due to this line, I didn't notice any changes in creating/updating badges. Can you please check if we still need it?